src/mmw/.jshintrc

```diff
@@ -6,6 +6,7 @@
     "browser": true,
     "curly": true,
     "eqeqeq": true,
+    "esnext": true,
```

Comment:
I'm not 100% sure we'd want to update this project to use ES6 syntax because I think some bits of it still may have to be transpiled to work across all browsers, and I think everything else here is ES5.